src/lisp.h

```diff
@@ -1276,6 +1276,9 @@ CDR_SAFE (Lisp_Object c)
 Lisp_Object Fsetcar(Lisp_Object, Lisp_Object);
 Lisp_Object Fsetcdr(Lisp_Object, Lisp_Object);
 Lisp_Object Fcar(Lisp_Object);
+Lisp_Object Fatom(Lisp_Object, Lisp_Object);
+Lisp_Object Flistp(Lisp_Object, Lisp_Object);
+Lisp_Object Fnlistp(Lisp_Object, Lisp_Object);
```

Comment:
These are functions of one argument, which is why the compile is failing on Travis.